src/Games/brain-cals.js

```diff
@@ -0,0 +1,31 @@
+import getRandomNumber from '../randomCount.js';
+import theCoreOfGames from '../index.js';
+
+const brainRulls = () => {
+  const descriptionOfGame = 'What is the result of the expression?';
+
+  const randomNumberOne = getRandomNumber(1, 10);
+  const randomNumberTwo = getRandomNumber(1, 10);
+  const operator = ['+', '-', '*'];
+  const randomOperator = getRandomNumber(0, 2);
+  const randomNumber = `${randomNumberOne} ${operator[randomOperator]} ${randomNumberTwo}`;
```

Comment:
Does this variable store a random number?

Reply:
I think the logic of getting a random operator needs to be moved to a separate function getRandomOperation

Reply:
ok, now it's 
const randomOperator = getRamdomOperation();
is it correct that i make this function inside mother roundGenerator function?